settings.gradle

```diff
@@ -66,14 +73,20 @@ if (file('local.properties').canRead()) {
 
 def calcVersion(buildconfig) {
     def local = gradle.rootProject.findProperty("local")
-    def version = new File(rootDir, 'version.txt').getText().trim()
 
-    if (gradle.rootProject.hasProperty("nightlyVersion")) {
+    if (gradle.hasProperty("mozconfig")) {
```

Comment:
Get something sane from a build in m-c - not clear how (or even if) we need a dedicated version string once landed.